secure_config.py

```
@@ -0,0 +1,214 @@
+def get_encryption_key() -> Optional[bytes]:
+    """Get the encryption key if it exists."""
+    key_file = Path.home() / ".cursor" / ".key"
+
+    if not key_file.exists():
+        return None
+
+    try:
+        with open(key_file, "rb") as f:
+            return f.read()
+    except (FileNotFoundError, PermissionError, IsADirectoryError, OSError) as e:
+        logger.exception("Failed to read encryption key from %s: %s", key_file, e)
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)